contentcuration/contentcuration/management/commands/exportchannel.py

```diff
@@ -211,13 +212,11 @@ def create_perseus_exercise(ccnode):
         logging.debug("Created exercise for {0} with checksum {1}".format(ccnode.title, assessment_file_obj.checksum))
 
 def create_perseus_zip(ccnode, write_to_path):
-    assessment_items = ccmodels.AssessmentItem.objects.filter(contentnode = ccnode)
+    assessment_items = ccmodels.AssessmentItem.objects.filter(contentnode = ccnode).order_by('order')
```

Comment:
standardize how exercise.json is generated to help with different checksum problem